notes/2017/2017-12-07.md

```diff
@@ -0,0 +1,84 @@
+# 07-Dec-2017 ESLint TSC Meeting Notes
+
+## Transcript
+
+https://gitter.im/eslint/tsc-meetings/archives/2017/12/07
+
+## Attending
+
+* Teddy Katz (@not-an-aardvark) - TSC
+* alberto (@alberto) - TSC
+* Kevin Partington (@platinumazure) - TSC
+* Kai Cataldo (@kaicataldo) - TSC
+* Brandon Mills (@btmills) - TSC
+* Ilya Volodin (@ilyavolodin) - TSC
+* Gyandeep Singh (@gyandeeps) - TSC
+* Toru Nagashima (@mysticatea) - TSC
+
+## Topics
+
+### [Proposal: Add URL for a rule's documentation to the API](https://github.com/eslint/eslint/issues/6582)
+
+* **TSC Summary**: This proposal suggests adding docs.uri/docs.url to meta property of each built-in rule with the link to rule's documentation.
+* **TSC Question**: Do we accept this proposal?
+* Q: What happens when docs change between versions?
+* A: Versioned links could solve this. How important is that vs how hard is it to implement? Let's leave versioned docs out of this particular change. We can address that later if necessary.
+* Q: Do we need to expose rule metadata in CLIEngine? We don't want people to start using the undocumented `cliEngine.linter.getRules()` method.
+* A: Add a `getRules()` instance method to `CLIEngine`. Currently it will pass through to the method on the internal `linter` instance.
+* :+1: from @ilyavolodin, @not-an-aardvark, @mysticatea, @btmills, @kaicataldo, @alberto, @gyandeeps
+* :+1: from @platinumazure with reservations about whether `CLIEngine` is the best place to put it.
```

Comment:
Apologies, I wasn't clear in the meeting: I'm :+1: to CLIEngine having some way to get the rules, I just wasn't 100% sure about proxying to `CLIEngine.linter.getRules()` because I wasn't sure what we wanted to do with Linter/CLIEngine integration in the future. That said, as long as we can preserve the API somehow I think this should be okay, so you can move me to unconditional :+1: if you like.